Trappist/src/Promact.Trappist.Web/wwwroot/app/tests/test-settings/test-launch-dialog.component.ts

```diff
@@ -1,8 +1,14 @@
 ﻿import { Component } from "@angular/core";
+import { Test } from "../tests.model";
 
 @Component({
     moduleId: module.id,
     selector: 'test-launch-dialog',
     templateUrl: "test-launch-dialog.html"
 })
-export class TestLaunchDialogComponent { }
+export class TestLaunchDialogComponent {
+    copiedContent: boolean = true;
+    settingObject: Test = new Test();
```

Comment:
initialization should go into constructor. But what is the requirement to initialize the class?